### PR TITLE
Feature translation #4

### LINKED ISSUE #4: Russian localization
1. set default timezone to Moscow
2. make gettext automatic Russian localization based on Accept-Language http header

### PR DESCRIPTION
**Pull request by avoevodin**
#4

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,6 @@
 *.pyc
 *.egg-info
 __pychache__/
-db.sqlite3
\ No newline at end of file
+db.sqlite3
+/mysite/locale/en/LC_MESSAGES/django.mo
+/mysite/locale/ru/LC_MESSAGES/django.mo
```
Comment (iliadmitriev) [inline .gitignore:12]: Need an empty line at the end of file

File: mysite/locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-03-17 09:13+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
```
Comment (iliadmitriev) [inline mysite/locale/en/LC_MESSAGES/django.po]: What language is this file for?

File: mysite/locale/ru/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-03-17 09:13+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
```
Comment (iliadmitriev) [inline mysite/locale/ru/LC_MESSAGES/django.po]: Language ?

File: mysite/polls/models.py [hunk 4]
```diff
@@ -1,11 +1,12 @@
 import datetime
 from django.db import models
 from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
 
 
 class Question(models.Model):
     question_text = models.CharField(max_length=200)
```
Comment (iliadmitriev) [inline mysite/polls/models.py:7]: Why this is not translated ?

**Review by iliadmitriev — CHANGES_REQUESTED** [review]
Need to fix

File: mysite/polls/models.py (hunk 4 above)
Comment (avoevodin) [reply mysite/polls/models.py:7]: Are you talking about the created questions?

Comment (iliadmitriev) [reply mysite/polls/models.py:7]: question_text field is not translated

File: mysite/polls/models.py [hunk 5]
```diff
@@ -17,6 +18,11 @@ def was_published_recently(self):
         now = timezone.now()
         return now >= self.pub_date >= timezone.now() \
             - datetime.timedelta(days=1)
+    was_published_recently.short_description = 'Опубликовано недавно'
```
Comment (iliadmitriev) [inline mysite/polls/models.py]: You should not add Russian text to code. Russian text should be in a translation file.

**Review by iliadmitriev — CHANGES_REQUESTED** [review]

File: mysite/polls/models.py [hunk 6]
```diff
@@ -29,3 +35,7 @@ def __str__(self):
             self.question, self.choice_text,
             self.id
         )
```
Comment (iliadmitriev) [inline mysite/polls/models.py:37]: lines 30-31 forgot to translate choice_text and votes fields

File: mysite/polls/templates/polls/results.html [hunk 7]
```diff
@@ -14,5 +16,5 @@ <h1>{{ question.question_text }}</h1>
     </ol>
     </div>
     <br>
-    <a class='btn btn-primary' href="{% url 'polls:detail' question.id %}">Vote again?</a>
+    <a class='btn btn-primary' href="{% url 'polls:detail' question.id %}">{% translate "Vote again?" %}</a>
 {% endblock %}
```
Comment (iliadmitriev) [inline mysite/polls/templates/polls/results.html]: no newline at the end of a file

**Review by iliadmitriev — CHANGES_REQUESTED** [review]

File: mysite/locale/en/LC_MESSAGES/django.po (hunk 2 above)
Comment (iliadmitriev) [reply mysite/locale/en/LC_MESSAGES/django.po]: still didn't specified the language code

File: mysite/locale/ru/LC_MESSAGES/django.po (hunk 3 above)
Comment (iliadmitriev) [reply mysite/locale/ru/LC_MESSAGES/django.po]: needed a language code

File: mysite/locale/en/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-03-17 10:22+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: enMIME-Version: 1.0\n"
```
Comment (iliadmitriev) [inline mysite/locale/en/LC_MESSAGES/django.po]: occasionally combined two lines in one

File: mysite/locale/ru/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-03-17 10:22+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: ruMIME-Version: 1.0\n"
```
Comment (iliadmitriev) [inline mysite/locale/ru/LC_MESSAGES/django.po]: occasionally combined two lines in one

**Review by iliadmitriev — CHANGES_REQUESTED** [review]

**Review by iliadmitriev — APPROVED** [review]

**Pull request merged**